prov/usnic/src/usdf_fabric.c

```diff
@@ -101,6 +102,10 @@ usdf_validate_hints(struct fi_info *hints, struct usd_device_attrs *dap)
 		return -FI_ENODATA;
 	}
 
+	if (FI_VERSION_LT(version, FI_VERSION(1, 1)) &&
+			(hints->mode & FI_MSG_PREFIX))
+		return -FI_ENODATA;
```

Comment:
Yeah it can, that was my mistake. That shouldn't be returning `-FI_ENODATA` that should be clearing it. I'll update.